WorkflowUI/Tests/ViewControllerDescriptionTests.swift

```diff
@@ -24,6 +24,10 @@
 
     fileprivate class BlankViewController: UIViewController {}
 
+    @objc fileprivate protocol MyProtocol {
```

Comment:
@kyleve curious if you have any ideas here: the test below only passes when this protocol has the `@objc` keyword. Otherwise, the `is` check fails.

Reply:
Nothing in particular, it's just one of the nuanced things about how `is` and swift vs objc interacts, unfortunately. I forget the specifics, but it's something about how protocol lookup and conformance works between the two languages and across modules, IIRC.